lib/archethic/mining/pending_transaction_validation.ex

```diff
@@ -79,6 +83,32 @@ defmodule Archethic.Mining.PendingTransactionValidation do
     end
   end
 
+  defp valid_not_exists(%Transaction{address: address}) do
+    storage_nodes = Election.chain_storage_nodes(address, P2P.authorized_and_available_nodes())
```

Comment:
As for other quorum_read, you should take only nearest nodes and locally available one.

But maybe this should be embedded directly in the quorum read function, as we always do it before. So we will avoid doing misstake on this selection and also don't write the same code many times

Reply:
:+1: